extras/command-completion/README.in

```diff
@@ -0,0 +1,7 @@
+This file is not moved to @bashcompdir@
+with the source installation.
+Please execute make install explicity from here to move this file to
+@bashcompdir@
+
+Similarly, use make uninstall to remove it from
+@bashcompdir@
```

Comment:
I think this is not true anymore since we have added the Makefile into configure.ac, so it will be installed and uninstalled as all other components. Maybe we should just remove this file.

Reply:
The way I understand it is that the `Makefile` and `README` files are generated by the configure step from the declarations in configure.ac, but the `Makefile` is not included from the `extras/Makefile` during the make step (`command-completion` not declared as a subdir), so the files are not included in dist files. Unless I am missing something in configure, I think that is also why I had to add the '.in'  and '.am' files explicitly to the dist files in `extras/Makefile`.

So as it is, the `extras/command-completion/Makefile` is still not run during make install. It could probably be. I tried to keep the same behaviour, as I don't know why the completion script should or should not be installed for a source installation. The Makefile is now an automake generated one instead of a short simple one, but it does basically the same thing.

Reply:
I think you are right. I assumed adding them to `configure.ac` was enough, but I forgot that the directory also needs to be added to some `Makefile.am` to be automatically processed.